src/test.rs

```diff
@@ -12,6 +12,7 @@ fn is_equal(lhs: &NodeRef, rhs: &NodeRef) -> bool {
         return false;
     }
 
+    // TODO: Do it more accurately.
```

Comment:
I think in practice we want to end up having a serializer. I don't think it should be too hard to match what bikeshed does, I can give it a poke next week if you want.

Reply:
Maybe we could do that when we meet issues related to the serializer, then we could validate our serializer with that case.